<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="qPushButtonReadFromFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Загрузить исходные данные из файла</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonSolver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Обработать данные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonClear">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonSaveToFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить обработанные данные в файл</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="qPlainTextEditMyArray">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelUslovie">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 0.6;
color: rgb(255, 255, 0)</string>
   </property>
   <property name="text">
    <string>Если после максимального элемента таблицы
расположена хотя бы одна единица,
то увеличить все положительные элементы
таблицы в два раза</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="qPushButtonArrayGenerator">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сгенерировать массив</string>
   </property>
  </widget>
  <widget class="QLabel" name="qLabelValues">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Максимальный элемент =</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
